docs/graphql/manual/auth/authorization/roles-variables.rst

```diff
@@ -34,8 +34,12 @@ Examples:
 Dynamic session variables
 -------------------------
 
-When you create a permission, or an access control rule, the permission rule itself needs access to some variables
-that are derived from the request itself. Let's refer to these as *session variables*.
+When you create a permission or an access control rule, the permission rule itself can use variables
+that come from the authentication service. Let's refer to these as *session variables*. 
+
+.. admonition:: ABAC
+
+  Session variables are analogous to *attributes* in a typical ABAC system.
```

Comment:
I think it would be good to expand the initialism here:

> Session variables are analogous to *attributes* in a typical attribute-based access control (ABAC) system.

Reply:
Also, this isn’t all that relevant to this particular change, but the previous sentence reads a bit strangely to me: “Let’s refer to these as session variables” sounds weirdly noncommittal! Why not write:

> Permission rules can also refer to *session variables*, which contain information returned from the authentication service.

Reply:
I don't mind what is present current but in case we are re-reading then, I would like to be more explicit: 

> Permission rules can also refer to *session variables*.  Session variables are key-value pairs  returned from the authentication service.

Reply:
I would move the note to the end of the section. It seems kind of odd in its current location _(bang in the middle of the definition and an example explaining it)_. Also I would hope to see a link to something speaking about abac attributes otherwise the entire section seems kind of unqualified

Agree with @lexi-lambda 's suggestion about the initial statement as well.

Reply:
Should it not be where it's defined?

Reply:
If its part of the deriniton, it doesnt need to be a note. Make it as a statement itself

Reply:
changed. I think it's fine to not link to any abac resource, dunno any authoritative resource anyway.

Reply:
I think a link to the Wikipedia article would probably be good enough, honestly.

Reply:
done !